src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -195,6 +216,21 @@ public void OnResponseHeadersStart()
                 }
             }
 
+            public void OnResponse100Continue()
+            {
+                // This is called when we receive complete set of headers with transient response code - like 100 Continue.
```

Comment:
"with transient response code - like 100 Continue".  This implies that this method could be called with other interim 1xx responses.  I don't think that is really true with this code.  So, I would rephrase this comment.

Reply:
this is more about server sending us unexpected response. When I look at HTTP/1 it consumes all 10x so I did it as well. We could possibly throw protocol error otherwise.

Reply:
So, this method is named "OnResponse100Continue()" but it can be called with any 1xx interim response received from the server?  If so, the method should be renamed to something like "OnInterimResponseReceived()".

Reply:
I checked and we do process all 1xx responses. However, this function is called only for 100 responses.   I updated the comment.